README.md

```diff
@@ -15,109 +15,130 @@ You got to the repository that collect all style configurations used by DevExtre
 
 ## TypeScript
 
-- **How to use**\
-    &nbsp; &nbsp; Add to your *.eslintrc*: `'extends': ['devextreme/typescript']`
+- **Usage**    
+    Add the following line to your *.eslintrc* file:
+    
+    ```javascript
+    'extends': ['devextreme/typescript']
+    ```
 
-- **Plugins**
+- **Required plugins**
   - [@typescript-eslint](https://github.com/typescript-eslint/typescript-eslint)
   
-- **Extends**
-  - `plugin:@typescript-eslint/recommended` [doc](https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin#recommended-configs)
-  - `airbnb-typescript` [doc](https://github.com/iamturns/eslint-config-airbnb-typescript)
+- **Extended plugins and configurations**
+  - [`plugin:@typescript-eslint/recommended`](https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin#recommended-configs)
+  - [`airbnb-typescript`](https://github.com/iamturns/eslint-config-airbnb-typescript)
   
 ## Spell Check
 
-Contains special words that should be ignored.
+Lists words that the spell check should ignore.
 
-- **How to use**\
-    &nbsp; &nbsp; Add to your *.eslintrc*: `"extends": ["devextreme/spell-check"]`
+- **Usage**
+    Add the following line to your *.eslintrc* file:
+    
+    ```javascript
+    "extends": ["devextreme/spell-check"]
+    ```
 
-- **Plugins**
+- **Required plugins**
   - [spellcheck](https://github.com/aotaduy/eslint-plugin-spellcheck)
 
 ## Renovation Declarations
 
-Use for the renovation declarations.
+- **Usage**    
+    Add the following line to your *.eslintrc* file:
+    
+    ```javascript
+    'extends': ['devextreme/renovation-declarations']
+    ```
+    
+- **Extended configurations**
+  - [`devextreme/typescript`](#typescript)
 
-- **How to use**\
-    &nbsp; &nbsp; Add to your *.eslintrc*: `'extends': ['devextreme/renovation-declarations']`
-- **Extends**
-  - `devextreme/typescript` [doc](#typescript)
+- **Overrides**
 
-- **Exceptions**
-  - Declarations used for generate target components in the development mode
+  - Allow importing devDependencies when in development mode:
 
-    ```javasxript
+    ```javascript
     'import/no-extraneous-dependencies': ['error', { devDependencies: true }]
     ```
 
-  - Declaration should contain two classes - component class and props class
+  - Allow two classes (component class and props class) in one file:
   
-    ```javasxript
+    ```javascript
     'max-classes-per-file': ['error', 2]
     ```
 
-  - Declaration is not a real React component and doesn't need it
+  - Do not require React. Declarations do not need it because they are not proper React components:
```

Comment:
```suggestion
  - Disable React dependency validation. Declarations do not require this dependency, since they are not proper React components. 
```